test/dt2js.test.js

```diff
@@ -477,3 +508,11 @@ describe('When property with name "items".', function () {
     return cb()
   })
 })
+describe('File with diferent kind of required types', function () {
+  it('should set required for `require: true` or without `required` types', function () {
+    const ramlData = fs.readFileSync(REQUIRED_TEST).toString()
+    const schema = dt2js.dt2js(ramlData, 'randomObject')
```

Comment:
Here's what's missing:
```suggestion
    dt2js.setBasePath('test/examples')
    const schema = dt2js.dt2js(ramlData, 'randomObject')
```